config.yaml

```diff
@@ -1,10 +1,21 @@
 # DB-related parameters
 database:
-  db_url: bolt://publicip:7687
-  db_username: username
-  db_password: password@dev
-  top_k: 1
-  neo4j_deeplink: "http://neodash.graphapp.io/xxxx"
+  embed:
+    url: bolt://publicip:7687
+    username: user
+    password: password@dev
+    top_k: 15
+  kg:
+    url: bolt://publicip_kg:7687
+    username: user
+    password: password@dev
+    order: 1
+    direction: both_ways
```

Comment:
Please add some comments here regardings options for different directions.

Reply:
Done